ui/views.py

```diff
@@ -117,13 +118,21 @@ def edit_vocabulary(request, repo_slug, vocab_slug):
 
 
 @login_required
+@permission_required_or_403(
```

Comment:
I haven't tried, but I think that the only difference is that the user will not be redirected to the login page but s/he will receive a 403 all the times because the anonymous user does not have any permission